app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -5,6 +5,11 @@ class ManageIQ::Providers::IbmCloud::Inventory::Parser::VPC < ManageIQ::Provider
 
   attr_reader :img_to_os
 
+  # Extensions to the list in operating_system.rb
+  VPC_OS_NAMES = [
+    ["linux_redhat",    %w(red-7 red-8)]
```

Comment:
Is this a VPC specific thing?  If not, I'd accept this change directly in core.

Reply:
OS names like `red-7-amd64`, `red-7-amd64-sap-applications` coming back from VPC.  Not sure if more odd names will come in the future, so thought having a local map to deal with them as they come could be useful.  But......I'm fine to move this into core if that's the right thing to do.

Reply:
So this would match if it were `redhat-7-abcd`:
```
>> OperatingSystem.normalize_os_name("redhat-7-amd64-sap-applications")
=> "linux_redhat"
```

We could fix this in the parser to normalize the `red-` names to `rhel` or `redhat`

Reply:
@agrare So that's basically what I added, just at the provider level.  Do you recommend moving this mapping to core?  Wasn't sure if `red-` would potentially conflict with something from another provider (say, Redfish).

Reply:
I don't think it would conflict with redfish, and as long as it was `"red-7"` and not just `"red-"` it would probably be fine.  I was thinking even simpler than what you have here though.

If we know that `red-*` should be `redhat-*` we could just change that before calling the core normalize_os_name without incurring the overhead of the `VPC_OS_NAMES` and the extra `normalize_vpc_os_name` method.

Something like:
```
os_name.sub!("red-", "redhat-") if name.start_with("red-")
OperatingSystem.normalize_os_name(os_name)
```

Reply:
Works for me, much simpler.  Don't anticipate too many OS names falling through the cracks of the core mapping.